config/config.yml

```diff
@@ -0,0 +1,9 @@
+keyrod:
+  fedcloud:
+    site: https://carach6.ics.muni.cz/
```

Comment:
Since this configuration file is used as a default when there are no other configuration files nor are options provided via CLI it doesn't really make sense to set defaults for values you can't really guess default for. Even wrose, this configuration file will go into gem and you're using address of one of our machines.
So if you're setting default for URLs use `localhost` (with appropriete path section if needed). That applies to `fedcloud->site` and `oidc->site` fields. And leave the fields empty if you can't guess the default value. That applies to `fedcloud->access_token`, `oidc->refresh_token` and probably `fedcloud->group` as well.